src/components/transactions/message-view/EditValidatorMessageDetails.vue

```diff
@@ -19,6 +19,7 @@
     </div>
     <div class="tx__content__right">
       <p class="amount">
+        <!-- TODO: FE should be network agnostic -->
         {{ fee.amount | atoms | prettyLong }}&nbsp;
```

Comment:
we can delete this whole message-view now ;)

Reply:
oh yeah, true. Like we discuss. But then we need to add some work in the API too, filter these transactions out, since otherwise these will be shown as empty divs

Reply:
I think I will do this in a separate PR: delete all the transactions components only useful for validators:
EditValidatorMessageDetails
UnjailMessageDetails
CreateValidatorMessageDetails
SetWithdrawAddressMessageDetails --> please, correct me if I'm wrong